README.md

```diff
@@ -255,8 +255,17 @@ Optional. Boolean. Default false. If true, it will print out debug messages. In
 
 Optional. Boolean. Default true. If false, will not log request and response body to Moesif.
 
+#### __`batch_size`__
+Optional. int, default 200, Maximum batch size when sending to Moesif.
+
+#### __`batch_max_time`__
+Optional. int in seconds Default 5.	This is the maximum wait time (approximately) before triggering flushing of the queue and sending to Moesif.
```

Comment:
@XinyuLuan let's change default to `2` seconds and update the code too.

Reply:
changed